.github/workflows/nix-managed-lints.yml

```
@@ -6,7 +6,14 @@
 
 name: "Project Lints"
 on:
+  # Contrary to the ./ci.yml workflow,
+  # this one is triggered _not_ triggered by the `push` events
```

Is this what was intended?
```suggestion
  # this one is _not_ triggered by the `push` events
```

---

Well, i tried to make the pr flox and the merge queue flow as similar as possible now
i.e. both fetch `refs/heads/main`.
I'm relatively confident that should be enough but yes, we'll only see after the queue is done.